bulk/eng_bulk_load-AssetRecord.csv

```diff
@@ -122,465 +128,481 @@ CGCON-CWFPCP-10302,A00847.1,Sensor,0,Coastal Wire-Following Profiler Controller,
+CGCON-NDCL16-00021,OL000101,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,0021,,,,,Woods Hole Oceanographic Institution
+CGCON-NDCL16-00022,OL000079,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,0022,,,,,Woods Hole Oceanographic Institution
 CGCON-NDCL16-00026,,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,26,,,,,
-CGCON-NDCL16-00030,,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,36,,,,,Woods Hole Oceanographic Institution
-CGCON-NDCL16-00033,,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,33,,,,,
+CGCON-NDCL16-00030,,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,0030,,,,,Woods Hole Oceanographic Institution
+CGCON-NDCL16-00033,,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,0033,,,,,Woods Hole Oceanographic Institution
 CGCON-NDCL16-00035,,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,35,,,,,
 CGCON-NDCL16-00036,,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,36,,,,,
+CGCON-NDCL16-00038,,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,38,,,,,
 CGCON-NDCL16-00043,,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,43,,,,,
 CGCON-NDCL26-00002,,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2,,,,,
-CGCON-NDCL26-00004,OL000488,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,4,,,,CP01CNSM-00001-DCL26,
-CGCON-NDCL26-00006,,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,6,,,,,
-CGCON-NDCL26-00007,OL000502,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,7,,,,CP01CNSM-00002-DCL26,
-CGCON-NDCL26-00009,,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,9,,,,CE07SHSM-00004-DCL26,
-CGCON-NDCL26-00010,,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,3703-00272-0010,,,,CE02SHSM-00005-DCL26,
-CGCON-NDCL26-00011,OL000571,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,11,,,,CP03ISSM-00001-DCL26,
-CGCON-NDCL26-00013,R00032,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,13,,,,CP03ISSM-00003-DCL26,
-CGCON-NDCL26-00014,OL000441,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,3703-00
```

Comment:
eng_bulk_load-AssetRecord_NEW_2019-05-17 spreadsheet has LEGACY_ASSET_UID of OL000583 - may be a copy/paste error (same OL# listed for 3 entries in a row in the csv)